### PR TITLE
drop Xserver/fb/fbmmx.[ch]

### PR DESCRIPTION
**Pull request by sunweaver**
This PR drops MMX code that upstream replaced by pixman_\* calls. We don't use pixman_\* calls, but we did not use MMX in Xserver/fb/ either (USE_MMX has never been defined, so far).

This PR drops the fbmmx.[ch] related code now, later on we should consider bumping the Xserver/fb code to latest X.org version status.

### CODE REVIEW

**Comment by sunweaver** [comment]
@Ionic: can you review this PR, please (#253). Thanks!

File: nx-X11/programs/Xserver/fb/Imakefile [hunk 1]
```diff
@@ -1,22 +1,3 @@
-/*
```
Comment (Ionic) [inline nx-X11/programs/Xserver/fb/Imakefile:1]: Leave out "(, yet)" in your commit message, you already included a yet earlier. :)

File: nx-X11/programs/Xserver/fb/fbpict.c [hunk 2]
```diff
@@ -901,11 +885,6 @@ fbComposite (CARD8      op,
 			switch (pDst->format) {
 			case PICT_r5g6b5:
 			case PICT_b5g6r5:
-#ifdef USE_MMX
-			    if (fbHaveMMX())
-				func = fbCompositeSolidMask_nx8x0565mmx;
-			    else
-#endif
 				func = fbCompositeSolidMask_nx8x0565;
```
Comment (Ionic) [inline nx-X11/programs/Xserver/fb/fbpict.c]: Theoretically, you could move the the call following the `USE_MMX` block one indentation level to the left, but that's really just an optical thing.

Would apply to all the following change blocks as well.

**Review by Ionic — COMMENTED** [review]
Otherwise looks good.

File: nx-X11/programs/Xserver/fb/Imakefile (hunk 1 above)
Comment (sunweaver) [reply nx-X11/programs/Xserver/fb/Imakefile:1]: > Leave out "(, yet)" in your commit message, you already included a yet earlier. :)

Done.

File: nx-X11/programs/Xserver/fb/fbpict.c (hunk 2 above)
Comment (sunweaver) [reply nx-X11/programs/Xserver/fb/fbpict.c]: > Theoretically, you could move the the call following the USE_MMX block one indentation level to the left, but that's really just an optical thing.

Done for all such blocks.

**Comment by sunweaver** [comment]
@Ionic: I force-pushed my changes to the PR branch. Ready for merging. Thanks!

**Pull request merged**